cost/budget_report_alerts/budget_report_alerts.pt

```diff
@@ -46,6 +46,12 @@ parameter "param_use_groups" do
   description "Use this parameter to specify how to group the data"
 end
 
+parameter "param_filter" do
+  type "list"
+  label "Group Filter by Dimension"
+  description "Filter by dimension=value like 'Cloud Vendor=AWS' 'Cloud Vendor Account=Account1,Account2' ; multiple filters with multiple values can be provided"
```

Comment:
Replace " ; multiple" with ". Multiple" to make this more clear and follow more standard grammar conventions.